capstone/src/main/resources/templates/manageevents.html

```diff
@@ -4,15 +4,19 @@
     <meta>
     <title>My Account</title>
     <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <script src="script.js"></script>
     <script src="user.js"></script>
+    <script src="event.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/timeago.js/4.0.2/timeago.min.js"></script>
```

Comment:
the `timeago` library might have to be loaded before your app's custom scripts if the scripts need to use `timeago`'s functionality.